ballista/rust/core/Cargo.toml

```diff
@@ -30,7 +30,7 @@ build = "build.rs"
 simd = ["datafusion/simd"]
 
 [dependencies]
-ahash = "0.7"
+ahash = { version = "0.7", default-features = false }
```

Comment:
At least `hashbrown` also depends on `ahash` by default.
So shouldn't we make that feature in `hashbrown` optional too?

Reply:
`hashbrown` sets `default-features = false` for `ahash`. https://github.com/rust-lang/hashbrown/blob/b3eaf32e608d1ec4c10963a4f495503d7f8a7ef5/Cargo.toml#L16

Reply:
Ok, that makes sense!